app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, favourite_dish: str,
+                 nickname: str,
+                 hummer_level: int) -> None:
+        super().__init__(favourite_dish, nickname)
+        self._hummer_level = hummer_level
```

Comment:
There is a typo in the attribute name `self._hummer_level`. It should be `self._hammer_level` to accurately describe the attribute and maintain consistency with standard terminology.